extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -73,11 +70,11 @@ export class AmpConsent extends AMP.BaseElement {
     /** @private {!Object<string, !ConsentUI>} */
     this.consentUI_ = map();
 
-    /** @private {!JsonObject} */
-    this.consentConfig_ = dict();
+    /** @private {?JsonObject} */
+    this.consentConfig_ = null;
```

Comment:
Why not just initialize the consent config here?

On line 120 you have const config = new ConsentConfig(this.element).getConfig();, which is then just set to `this.consentConfig_` 😄

Reply:
I think maybe the confusion is around naming? (at least for me) I would expect `this.consentConfig_` to be set to the result of `new ConsentConfig(this.element).getConfig()` since they are named the same, but it is actually `config['consents']`. Maybe something like `this.consents_` would help?